components/registry-facade/pkg/registry/blob.go

```diff
@@ -359,8 +359,7 @@ func (sbs ipfsBlobSource) HasBlob(ctx context.Context, spec *api.ImageSpec, dgst
 func (sbs ipfsBlobSource) GetBlob(ctx context.Context, spec *api.ImageSpec, dgst digest.Digest) (dontCache bool, mediaType string, url string, data io.ReadCloser, err error) {
 	log := log.WithField("digest", dgst)
 
-	var ipfsCID string
-	ipfsCID, err = sbs.source.Redis.Get(ctx, dgst.String()).Result()
+	ipfsCID, err := sbs.source.Redis.Get(ctx, dgst.String()).Result()
```

Comment:
This is intended, because we declare the err variable at line 359.
Otherwise, we can't catch this line's error and update to line 359 err variable.

Reply:
> Otherwise, we can't catch this line's error and update to line 359 err variable.

Oh, I didn't know that. So you are saying that we can't use err for return value?

Reply:
Oops, sorry my bad, I did not see at line 365, we set the error as `err = distv2.ErrorCodeBlobUnknown`.